apps/transport/lib/jobs/gtfs_diff.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Transport.Jobs.GtfsDiff do
```

Comment:
```suggestion
defmodule Transport.Jobs.GTFSDiff do
```
Me semble qu'on le met en majuscules ailleurs dans la code base ?

Reply:
Y'a pas mal de contre exemples, `SingleGtfsToGeojsonConverterJob`, `Mix.Tasks.Decode.GtfsRt`, `GtfsValidator`, ...